vllm/v1/metrics/loggers.py

```diff
@@ -353,13 +359,19 @@ def __init__(self, vllm_config: VllmConfig, engine_index: int = 0):
                         self.labelname_max_lora,
                         self.labelname_waiting_lora_adapters,
                         self.labelname_running_lora_adapters,
-                    ])
+                    ],
+                    multiprocess_mode="livemostrecent"
```

Comment:
Is this lora_info gauge another potentially problematic one? Since it's updated via IteratonStats and not SchedulerStats ... So I'm not sure "livemostrecent" is the right thing to use here.

I haven't looked closely at what this metric / how it is computed, maybe "sum" would fit better. I have a feeling though even that might not be correct, because it may be counting e.g. the number of unique lora adapters across the running requests and so it's not really possible to just combine the separate counts when the requests from a given engine are partitioned.

If there's not an easy answer we could include this in the list that we document as not being correct when mutli-api-servers are in play.

Reply:
In v0 world, this was livemostrecent. I assumed it was intentional, so I kept it. 
https://github.com/vllm-project/vllm/blob/main/vllm/engine/metrics.py#L80

It is potentially one of those problematic ones. But supporting lora metrics even falls lower in priority than the other one. So we may as well just record it in the docs. I'd keep as is since it's coming from historical context anyways.

Reply:
In V0 I don't think were were ever logging the same metrics in multiple places, so the aggregation mode was probably irrelevant. I do think "sum" is probably slightly less bad here. But probably we should just disable this metric when there are multiple API servers since I think the values will just be wrong (and can document of course).

Reply:
I made it sum. We can simply document for now.